articles/aks/wasmcloud.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Use wasmCloud on Azure Kubernetes Service (AKS)
```

Comment:
Just to be consistent with SpinKube article: https://learn.microsoft.com/en-us/azure/aks/deploy-spinkube

I would suggest the title to be "Deploy wasmCloud to Azure Kubernetes Service (AKS) to run distributed WebAssembly (Wasm) workloads"